Add robocup incheon report (#6)
tu-darmstadt-ros-pkg/teamhector.de: _competitions/2026_robocup.md

---
name: RoboCup 2026 Incheon (Korea)
image: 2026_robocup.jpg
date: 2026-06-30
competition: RoboCup World Championship
---

> At the RoboCup Rescue World Championship 2026 in Incheon, Korea, Team Hector achieved advancement into the semi-finals among a record number of 23 competing international teams.

Reviewer: ```suggestion
At the RoboCup Rescue World Championship 2026 in Incheon, Korea, Team Hector advanced to the semi-finals among a record number of 23 competing international teams.
```

> We managed to demonstrate a significant improvement in mobility results by employing a rework of our belt profiles as well as the base plate. On the one hand, Athena's turning ability was improved by a reduction of floor contact, and consequently friction, for this type of motion. On the other hand, the overall force-to-ground transmission was enhanced by improved traction of the main belt profiles as well as an anti-friction base plate.

Reviewer: ```suggestion
We achieved a significant improvement in mobility after optimizing the belt profiles and adding an anti-friction base plate. In particular, Athena's turning performance improved due to a more efficient distribution of ground contact.
Extensive work on hardware robustness and low-level software reliability also paid off, as Athena experienced zero hardware faults across all RoboCup runs.
```
